packages/modals/src/containers/FocusJailContainer.spec.js

```diff
@@ -96,27 +95,44 @@ describe('FocusJailContainer', () => {
           </FocusJailContainer>
         );
         wrapper.simulate('keydown', { keyCode: KEY_CODES.TAB });
-        jest.runOnlyPendingTimers();
 
         expect(focusElementSpy).toHaveBeenCalledTimes(2);
         expect(focusElementSpy).toHaveBeenLastCalledWith(findContainer(wrapper).getDOMNode());
       });
 
       it('focuses first element if tab key is pressed', () => {
         wrapper.simulate('keydown', { keyCode: KEY_CODES.TAB });
-        jest.runOnlyPendingTimers();
 
-        expect(focusElementSpy).toHaveBeenCalledTimes(2);
-        expect(focusElementSpy).toHaveBeenLastCalledWith(findButton(wrapper).getDOMNode());
+        expect(focusElementSpy).toHaveBeenCalledTimes(1);
+        expect(focusElementSpy).toHaveBeenLastCalledWith(findContainer(wrapper).getDOMNode());
       });
 
       it('focuses last element if tab and shift key is pressed', () => {
         wrapper.simulate('keydown', { keyCode: KEY_CODES.TAB, shiftKey: true });
-        jest.runOnlyPendingTimers();
 
         expect(focusElementSpy).toHaveBeenCalledTimes(2);
         expect(focusElementSpy).toHaveBeenLastCalledWith(findInput(wrapper).getDOMNode());
       });
+
+      it("doesn't intercept tab key if not the first or last tabbable item", () => {
```

Comment:
Could you run a `yarn format`? Not sure why the `"` isn't getting caught in the linting

Reply:
heh yarn format changed that to double quotes I originally had an escape `\'` with single quotes

Reply:
Go ahead and merge, then! (Or use a curly single quote ;)